<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Strict//EN">
<html style="direction: ltr;" lang="en"><head> <meta http-equiv="content-type" content="text/html; CHARSET=windows-1255">


	<title>David Burstein's homepage</title>
	<meta name="author" content="David Dudu Burstein">
	<META NAME="keywords"
		  CONTENT="David,Burstein,Bioinfo,bioinformatics,bioinformatic,legionella,machine,learning,דודו,ודוד,דוד,בורשטיין,ןייטשרוב">
	<meta name="description" content="Well this is my academic
									  homepage from the time I've been a PhD
									  student in Tel Aviv university,
									  Check out my roommate's homepage,
									  he is really something... = )">
	<style title="DBsStylesheet" type="text/css">
  p { margin: 1pt 0pt 0pt 1em;
    }

  a:link { text-decoration: none;
    color: rgb(0, 0, 153);
    }

  a:visited { text-decoration: none;
    color: rgb(0, 0, 153);
    }

  a:hover { text-decoration: none;
    color: rgb(51, 51, 255);
    }

  .DBsBox { border: 1px inset rgb(51, 0, 0);
    margin: 1%;
    padding: 0.5%;
    background-color: rgb(255, 255, 255);
    text-align: left;
    vertical-align: top;
    }

  .DBsTable { vertical-ali
  gn: top;
    }

  .DBsOutlineTable { margin: 1px;
    width: 100%;
    }

  .DBquotation { font-family: georgia,'palatino linotype','book antiqua',palatino,'times new roman',times,serif;
    }

  .DBsText { color: rgb(128, 0, 0);
    font-size: small;
    font-family: Arial,Helvetica,sans-serif;
    line-height: 1.5em;
    }

  .DBsPicture { border: 1px outset rgb(51, 0, 0);
    margin-top: 4px;
    }

  .DBsHeading { margin: 0.5em 0pt 0.4em 5px;
    font-family: Times New Roman,Times,serif;
    font-weight: lighter;
    font-style: italic;
    font-size: xx-large;
    }


</style></head>
<body style="background-image: url(pics/background4.jpg);" class="DBsText">
	<p class="DBsHeading">
	David Burstein's Homepage
	</p>
	<table>
	<tbody>
		<tr>

			<td style="width: 70%;" class="DBsBox">

				<table>
				<tbody class="DBsTable">
					<tr>

						<td>
							<img class="DBsPicture" alt="David Dudu Burstein" title="= )" src="pics/theGrudge.jpg">
						</td>

						<td>
							<p>
								Hi, I'm currently a joint postdoc in the
								labs of 
								<a href="http://geomicrobiology.berkeley.edu/">
								Jill Banfield</a> 
								and <a href="http://rna.berkeley.edu/">
								Jennifer Doudna</a>
								at UC Berkeley.
								<br>
								I've completed my Ph.D. studies under the supervision of
								<a href="http://www.tau.ac.il/~talp/"> Prof. Tal Pupko</a>,
								in the
								<a href="http://www.tau.ac.il/lifesci/">George S. Wise Faculty of Life Sciences</a>,
								<a href="http://www.tau.ac.il">Tel Aviv University</a>
								<br>
								<br>
								I was also a proud fellow of the <a href="http://safrabio.cs.tau.ac.il/index.htm">  Edmond J. Safra Bioinformatics Program</a>.
								<br>
								<br>
								E-mail:
								<a href="mailto:davidbur@berkeley">davidbur@berkeley.edu</a>
								<br>
							</p>
						</td>
					</tr>
					</tbody>
				</table>
			</td>

			<td></td>

			<td style="width: 30%;"class="DBsBox">
				<p>
					<br>
					<b>Research interests:</b>
					<br>
					</p><p>
					In this stage of my academic life,
					I find these topics of great interest:
					<br>
					<br>
					- Microbial communities
					<br>
					- Metagenomics
					<br>
					- Function prediction
					<br>
					- Bacterial genomics
					<br>
					- Machine learning
					<br>
					- Phage-host interaction
					<br>
					- Molecular evolution and phylogeny
					<br>
					<br>
				</p>
				<hr style="color: rgb(51, 0, 0);">
				<p>
					<br>
					<a href="DavidBursteinCV.pdf"> My CV </a>
				</p>
			</td>
		</tr>

		</tbody>
	</table>


<table class="DBsOutlineTable" cellspacing="2">
	<tbody>
		<tr>
			<td class="DBsBox">

				<table>
				<tbody class="DBsTable">
					<tr>
						<td style="vertical-align: top;">
							<img class="DBsPicture" src="pics/Saba.jpg">
						</td>

						<td>
							<p>
								<b>Publications</b>
								<br><br>
								20. Probst A.J., Castelle C.J., Singh A., Brown
								C.T., Anantharaman K., Sharon I., Hug L.A.,
								<u>Burstein D.</u>, Emerson J.B., Thomas BC.,
								and Banfield J.F.
								Genomic resolution of a cold subsurface aquifer community provides
								metabolic insights for novel microbes
								adapted to high CO2 concentrations.
								<i>Environmental Microbiology</i> doi:
								10.1111/1462-2920.13362. [Epub ahead of print] (2016)
								<br><br>
								19. <u>Burstein D.</u>, Sun C.L., Brown C.T., Sharon I.,
								Anantharaman K., Probst A.J., Thomas B.C.,
								and Banfield J.F.
								Major bacterial lineages are essentially
								devoid of CRISPR-Cas viral defence
								systems.
								<i>Nature Communications</i> 7:10613 (2016)
								<br><br>
								18. Anantharaman K., Brown C.T., <u>Burstein D.</u>,
								Castelle C.J., Probst A.J., Thomas B.C.,
								Williams K.H., and Banfield J.F.
								Analysis of five complete genome
								sequences for members of the class
								Peribacteria in the recently recognized
								Peregrinibacteria bacterial phylum
								<i>PeerJ</i> 4, e1607 (2016)
								<br><br>
								17. <u>Burstein D.</u>, Amaro F., Zusman T., Lifshitz
								Z., Cohen O., Gilbert J.A., Pupko T., Shuman
								H.A., and Segal G.
								Genomic analysis of 38 <i>Legionella</i>
								species identifies large and diverse
								effector repertoires.
								<i>Nature Genetics</i> 48(2):167-75 (2016)
								<br><br>
								16. Teper D.*, <u>Burstein D.</u>*, Salomon
								D., Gershovitz M., Pupko T., and Sessa G.
								Identification of novel <i>Xanthomonas
								euvesicatoria</i> type III effector proteins
								by a machine-learning approach.
								<i>Molecular Plant Pathology</i>
								17(3):398-411 (2015)
								<br><br>
								15. <u>Burstein D.</u>*, Satanower S.*, Simovitch M.*, Belnik Y., Zehavi M., Yerushalmi G.,
								Ben-Aroya S., Pupko T., and Banin E.
								Novel Type III Effectors in <i>Pseudomonas
								aeruginosa</i>.
								<i>mBio</i> 6 (2), e00161-154 (2015)
								<br>
								<br>
								14. Sharon I., Kertesz M., Hug L.A., Pushkarev D., Blauwkamp T.A., Castelle C.J.,
								Amirebrahimi M., Thomas B.C., <u>Burstein D.</u>,
								Tringe S.G., Williams K.H., and Banfield
								J.F.
								Accurate, multi-kb reads resolve complex
								populations and detect rare
								microorganisms.
								<i>Genome research</i> 25 (4), 534-54318
								(2015)
								<br>
								<br>
								13. Lifshitz Z.,<u>Burstein D.</u>, Schwartz K., Shuman H.A., Pupko T.,
								and Segal G.
								Identification of Novel <i>Coxiella
								burnetii</i> Icm/Dot Effectors and Genetic
								Analysis of Their Involvement in
								Modulating a Mitogen-Activated Protein
								Kinase Pathway.
								<i>Infection and immunity</i> 82 (9),
								3740-3752 (2014)
								<br>
								<br>
								12. Yosef I., Shitrit D., Goren
								M.G., <u>Burstein D.</u>, Pupko T., and
								Qimron U.
								DNA motifs determining the efficiency of
								protospacer adaptation into the
								Escherichia coli CRISPR
								array. <i>Proceedings of the National
								Academy of Sciences USA</i> 110(35):
								14396--14401 (2013)
								<br>
								<br>
								11. Cohen O., Ashkenazy H., Karin E.L., <u>Burstein D.</u>, and Pupko
								T. 
								CoPAP: Co-evolution of Presence-Absence Patterns. <i>Nucleic Acids
								Research</i> (Web Server issue):
								W232-237 (2013)
								<br>
								<br>
								10. Lifshitz Z.*, <U>Burstein D.*</U>, Peeri
								M., Zusman T., Schwartz K., Shuman
								H. A., Pupko T., and Segal G. 
								Computational modeling and experimental validation 
								of the <i>Legionella</i> and <i>Coxiella</i>
								virulence-related Type IVB secretion
								signal. <i> Proceedings of the National
								Academy of Sciences USA</i>. 
								110(8):E707-715 (2013).
								<br>
								<br>
								9. Cohen, O., Ashkenazy, H., <U>Burstein, D.</U>,
								and Pupko, T. Uncovering the
								co-evolutionary network among
								prokaryotic genes. <i>Bioinformatics</i>.
								<b>28</b> ECCB 2012:i389-i394 (2012)
								<br>
								<br>
								8. <U>Burstein, D.</U>*, Gould, S.B.*, Zimorski, V., Klosges, T., Kiosse, F., Major, 
								P., Martin, W., Pupko, T., Dagan, T. A machine-learning approach to 
								identify hydrogenosomal proteins in <i>Trichomonas vaginalis</i>. <i>Eukaryotic 
								Cell</i>. <B>11</B>:217-228 (2012)
								<a href="http://www.tau.ac.il/~talp/publications/BursteinEukaryotic Cell2012.pdf" target="_blank">
								[pdf]</a> <a href="http://dx.doi.org/10.1128/EC.05225-11" target="_blank">[abs]</a>
								<br>								
								<br>
								7. Amit, M., Donyo, M., Hollander, D.,
								Goren, Aa., Kim, E., Gelfman, S.,
								Lev-Maor, G., <U>Burstein, D.</U>, Schwartz,
								S., Postolsky, B., Pupko, T., and Ast,
								G. Differential GC content
								between exons and introns establishes
								distinct strategies of splice-site
								recognition. <i>Cell Reports</i>. <b>1</b>(5):543-556 (2012)
								<a href="http://www.tau.ac.il/~talp/publications/CellReportsGC_splicing.pdf" target="_blank">
								[pdf]</a> <a href="http://dx.doi.org/10.1016/j.celrep.2012.03.013" target="_blank">[abs]</a>
								<br>
								<br>
								6. Gelfman, S., <U>Burstein, D.</U>, Penn, O., Schwartz, S., Pupko, T., and Ast, G. 
								Changes in exon-intron structure during vertebrate evolution affect the 
								splicing pattern of exons. <i>Genome
								Research</i>. <B>22</B>(1):35-50 (2012)
								<a href="http://www.tau.ac.il/~talp/publications/exonIntronVertebrateEvol.pdf" target="_blank">
								[pdf]</a> <a href="http://dx.doi.org/10.1101/gr.119834.110" target="_blank">[abs]</a>
								<br>
								<br>
								5. Barzel, A., Privman, E., Peeri, M., Naor, A., Shachar, E., <U>Burstein, D.</U>, 
								Lazary, R., Gophna, U., Pupko, T., and Kupiec, M. Native homing 
								endonucleases can target conserved genes in humans and in animal models. 
								<i>Nucleic Acids
								Research</i>. <b>39</b>(15):6646-6659 (2011)
								<a href="http://www.tau.ac.il/~talp/publications/HEGs.pdf" target="_blank">[pdf]</a> 
								<a href="http://dx.doi.org/10.1093/nar/gkr242" target="_blank">[abs]</a>
								<br>
								<br>
								4. <U>Burstein, D.</U>, Zusman, T., Degtyar, E., Viner, R., Segal, G., and Pupko, T.
								Genome-scale identification of <i>Legionella pneumophila</i> effectors using a
								machine learning approach. <i>PLoS
								Pathogens</i>. <b>5</b>(7): e1000508 (2009)
								<a href="http://www.tau.ac.il/~talp/publications/LegionellaPlos.pdf">[pdf]</a>
								 
								<a href="http://dx.doi.org/10.1371/journal.ppat.1000508">[abs]</a>
								<br>
								<br>
								3. Schwartz, S., Silva, J., <U>Burstein, D.</U>, Pupko, T., Eyras, E., and Ast, G.
								Large scale comparative analysis of splicing signals and their
								corresponding splicing factors in
								eukaryotes. <i>Genome
								Research</i>. <b>18</b>(1):88-103
								(2008) <a href="http://www.tau.ac.il/~talp/publications/ancestralPSSM.pdf">	[pdf]</a>
								 
								<a href="http://dx.doi.org/10.1101/gr.6818908">[abs]</a>
								<br>
								<br>
								2. Ulitsky, I., <U>Burstein, D.</U>, Tuller, T., and Chor, B.
								The ACS approach to phylogenomic reconstruction.
								<i>Journal of Computational Biology</i>.
								<b>13</b>(2):336-50 (2006)
								<a href="http://www.liebertonline.com/doi/pdf/10.1089/cmb.2006.13.336">[pdf]</a>
								<br>
								<br>
								1. <U>Burstein D.</U>, Ulitsky I., Tuller T. and Chor B.
								Information theoretic approaches to whole genome phylogenomics.
								Proceedings of the ninth annual international conference on
								research in computational molecular biology, <i>RECOMB 2005</i>.
								pp.283-295 (2005)
								<a href="http://www.springerlink.com/content/ju4wejgc1u2flmc4/"> [abs]</a>
								<br>
								<br>
								<small>* These authors contributed equally</small>
							</p>
						</td>
					</tr>
				</tbody>
				</table>
			</td>
		</tr>
	</tbody>
</table>

<table class="DBsOutlineTable" cellspacing="2" width ="50%">
	<tbody>
		<tr>
			<td class="DBsBox">

				<table>
				<tbody class="DBsTable">
					<tr>
						<td style="vertical-align: top;">
							<img class="DBsPicture" src="pics/Petals.jpg">
						</td>

						<td>
							<p>
								<b>Academic Prizes and Awards</b>
							</p>
							<br>
								<table>
								<tbody class="DBsTable">
                                    <tr>
										<td><p>
										2014
										</p></td>
										<td><p>
										<b> EMBO Long-Term postdoctoral
										fellowship</b> 
										</td></p>
									</tr>	
<tr>
										<td><p>
										2013
										</p></td>
										<td><p>
										<b> Rothschild postdoctoral
										fellowship</b>
										</td></p>
									</tr>	
                                    <tr>
										<td><p>
										2012
										</p></td>
										<td><p>
										Training scholarship for Ph.D. students - The Aharon Katzir Center,
										Weizmann Institute of Science
										</td></p>
									</tr>	
									<tr>
										<td><p>
											2012
										</p></td>
										<td><p>
											<B>Prof. Willy Hirsch Prize for original research work conducted in the field of microbiology, awarded the Israel Society for Microbiology (ISM)</B>
										</td></p>
									</tr>
                                    <tr>
										<td><p>
										2012
										</p></td>
										<td><p>
										Outstanding poster award - the
										20th Annual International
										Conference on Intelligent Systems for Molecular Biology (ISMB), Long Beach, California
										</td></p>
									</tr>	
									<tr>
										<td><p>
											2012
										</p></td>
										<td><p>
											Outstanding poster award - The 14th Israeli Bioinformatics Symposium, Hebrew University of Jerusalem
										</td></p>
									</tr>	
									<tr>
										<td><p>
											2012
										</p></td>
										<td><p>
											Best poster award - Seventh Edmond J. Safra Bioinformatics Retreat
										</td></p>
									</tr>
									<tr>
										<td><p>
											2012
										</p></td>
										<td><p>
											Award from the Tel-Aviv University graduate school for excellent achievements in teaching, research and studies
										</td></p>
									</tr>	
									<tr>
										<td><p>
											2012
										</p></td>
										<td><p>
											The Anat Krauskopf Fund Travel Award
										</td></p>
									</tr>
									<tr>
										<td><p>
											2010
										</p></td>
										<td><p>
											Best poster award - The Annual Meeting of the Israeli Society for Microbiology (ISM)
										</td></p>
									</tr>
									<tr>
										<td><p>
											2010
										</p></td>
										<td><p>
											Best poster award - Fifth Edmond J. Safra Bioinformatics Retreat
										</td></p>
									</tr>
									<tr>
										<td><p>
											2009 - 2013
										</p></td>
										<td><p>
											<B>Excellence Scholarship from the Converging Technologies Program  of the Israeli Council for Higher Education</B>
										</td></p>
									</tr>
									<tr>
										<td><p>
											2009
										</p></td>
										<td><p>
											Best poster award - The 12th Israeli Bioinformatics Symposium, Weizmann Institute of Science
										</td></p>
									</tr>
									<tr>
										<td><p>
											2008
										</p></td>
										<td><p>
											<B>Wolf Foundation Award for
											excellent M.Sc. students</B>
											(nation-wide prize) 
										</td></p>
									</tr>
									<tr>
										<td><p>
											2008
										</p></td>
										<td><p>
											Award from the Tel-Aviv University graduate school for excellent achievements in teaching, research and studies
										</td></p>
									</tr>
									<tr>
										<td><p>
											2006 - 2009
										</p></td>
										<td><p>
											Scholarship for excellent students from the Edmond J. Safra bioinformatics program
										</td></p>
									</tr>
									<tr>
										<td><p>
											2003 - 2004
										</p></td>
										<td><p>
											Exact Science Dean's honour list
										</td></p>
									</tr>
									<tr>
										<td><p>
											2002, 2004
										</p></td>
										<td><p>
											Life Science Dean's honour list
										</td></p>
									</tr>
								</tbody>
								</table>
						</td>
					</tr>
				</tbody>
				</table>
			</td>
		</tr>
	</tbody>
</table>
	<table class="DBsOutlineTable" cellspacing="2">
	<tbody>
		<tr>
		<td style="width: 100%;" class="DBsBox">

										<table>
										<tbody class="DBsTable">
											<tr>
												<td style="vertical-align: top;">
													<img class="DBsPicture" src="pics/LadyBug.jpg">
												</td>

												<td>
													<p>
														<b>Past teaching</b>
														<br>
														<br>
														Perl programming
														for biology (instructor):
														<br>
														<a href="http://ibis.tau.ac.il/perluser/2012/">autumn 2012</a>
														,
														<a href="http://ibis.tau.ac.il/perluser/2011/">autumn 2011</a>
														,
														<a href="http://ibis.tau.ac.il/perluser/2010/">autumn 2010</a>
														,
														<a href="http://ibis.tau.ac.il/perluser/tashsat/">spring 2009</a>
														,
														<a href="http://ibis.tau.ac.il/twiki/bin/view/Bioinformatics/PERL2011">workshop 2010</a>
														<br>
														Introduction to
														Bioinformatics
														(teacher assistant):
														<br>
														<a href="http://ibis.tau.ac.il/intro_bioinfo/Previous/07-08">autumn 2007-2008</a>
														<br>
														Biological
														Sequences
														Analysis (in
														MTA, instuctor):
														<br>
														<a href="http://ibis.tau.ac.il/intro_bioinfo/mta/2008/index.spring08.html">spring 2008</a>
														,
														<a href="http://ibis.tau.ac.il/intro_bioinfo/mta/2009/">spring 2009</a>
														<br>

													</p>
												</td>
											</tr>
										</tbody>
										</table>
			</td>
		</tr>

	</tbody>
	</table>

<table class="DBsOutlineTable" cellspacing="2">
	<tbody>
		<tr>
			<td class="DBsBox">

				<table>
				<tbody class="DBsTable">
					<tr>
						<td style="vertical-align: top;">
							<img class="DBsPicture" src="pics/antLeaf.jpg">
						</td>

						<td>
							<p>
								<b>Some Academic tools / info</b>
								<br>

								<a href="makefile.pptx">makefile.pptx</a>
								- a short tutorial for makefiles
								<br>
								<a href="makeCompileFeaturesByPtts.makefile">makeCompileFeaturesByPtts.makefile</a> - currently my favorite makefile. It is a makefile I recently wrote that orchestrates exection of numerous perl script files
										<br>
										<a href="http://www.cs.tau.ac.il/~davidbur/CloneMe/">Clone Me!</a> little aid for cloning Legionella ORFs
										<br>
										<a href="http://www.cs.tau.ac.il/~davidbur/OligoBarzel/">OligoBarzel</a>
										Split sequence into k-mer for
										oligo-templated polymerization
										<br>
										<a href="http://db.tt/l3tgQh">An Eclipse bundle</a> includes also Perl (+ EPIC + BioPerl?) + R (+StatET)
										<br>
										<br>
										</p>
						</td>
					</tr>
				</tbody>
				</table>
			</td>
		</tr>
	</tbody>
</table>
<table class="DBsOutlineTable" cellspacing="2">
	<tbody>
		<tr>
			<td class="DBsBox">

				<table>
				<tbody class="DBsTable">
					<tr>
						<td style="vertical-align: top;">
							<img class="DBsPicture" src="pics/Ddelight.jpg">
						</td>

						<td>
							<p>
								<b>Some personal stuff</b>
								<br>
										</p><p>
										<a href="http://www.tau.ac.il/~davidbur/Liko">My roomate's homepage ^ ^</a>
										<br>
										<a href="pics/working with j.jpg">Our new cute roomate</a>
										<br>
										My beloved one &nbsp;&nbsp; <a href="pics/IrisTurtle.JPG">+ baby turtle</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="pics/IrisYam.jpg">+ me</a>
										<br>
										<a href="pics/Shomit.jpg">My artist sister Salom&eacute; </a>
										<br>
										<a href="pics/ZVnMik.jpg">My baby sister Avigail (aka ZV) and her Michael </a>
										<br>
										<br>
							</p>
						</td>
					</tr>
				</tbody>
				</table>
			</td>
		</tr>
	</tbody>
</table>
<center>
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-8883082-2");
pageTracker._trackPageview();
} catch(err) {}</script>
</body></html>
